databases/week2/DB_W2_HW.sql

```diff
@@ -0,0 +1,37 @@
+-- 1. Add a task with the these attributes: title, description, created, updated, dueDate, statusID, userID
+INSERT INTO task 
+(title, description, created, updated, due_Date, status_id, user_id)
+VALUES 
+('Make a new task' ,
+'Make a new task in task table',
+NOW(),
+NOW(),
+NOW() + INTERVAL 5 DAY,
+1,
+1);
+
+-- 2. Change the title of a task with these attributes: taskID, newTitle
+UPDATE task
+SET description = 'Change the title of a task'
+WHERE (id = 37);
+
+-- 3. Change the task due date with these attributes: taskID, newDueDate
+UPDATE task
+SET due_date = due_date + INTERVAL 20 DAY
+WHERE (id = 37);
+-- I intentionally were trying to see if I can add for example 20 days to the old due_date.
```

Comment:
Cool :+1: Again using INTERVAL can be useful

Reply:
Oh by the way: You don't need the parenthesis in the WHERE clause. It is usually only necessary in certain cases when combining AND and OR

Reply:
You're right. It is just me being conservative and trying to see too far in the future ;)